rest-api/dao/participant_summary_dao.py

```diff
@@ -201,9 +204,12 @@ def update_from_biobank_stored_samples(self, participant_id=None):
     If participant_id is provided, only that participant will have their summary updated."""
     baseline_tests_sql, baseline_tests_params = get_sql_and_params_for_array(
         config.getSettingList(config.BASELINE_SAMPLE_TEST_CODES), 'baseline')
+    self._back_fix_new_codes(baseline_tests_params)
```

Comment:
This seems wrong. Where baseline_tests_sql is used below, we *do* want to count 1PS08, 1PST8, 1SS08, and 1SST8. So I don't think we need to call this method.